### DISCUSSION TITLE
Touch in pygame/pygbag

### DISCUSSION THREAD

**Discussion by larstuff** [pygame-web/pygbag#89] [Q&A]
Hi. I saw a thread og pygbag and touch posted on reddit, and found a beautiful demo where it was working, this is the link to the demo:
https://pygame-web.github.io/showroom/pygame-scripts/org.pygame.touchpong.html 

I have a couple of questions: 
1) How do You make pygame autostart without the usual 'game ready' clicker?
2) How do You work with touch in pygbag/pygame?
Thanks

**Comment by pmp-p** [comment]
for 1 use prepend `--ume_block 0` on pygbag command line , for 2 you can have a look to  [pygame touch interface documentation](https://pyga.me/docs/ref/event.html) or some pygame android games.

**Reply by larstuff** [reply]
Thanks a lot!

I saw the --ume_block 0 thing in another post You made, and already implemented that.

When it comes to touch, I think I'll leave that for just now, but thanks for the link. It seems both Android and
iPhone (possibly also iPad) work well with mousedown (which really is fingerdown) and mouseup, so one can use
a combination of mouse_pos and mousebuttonup to capture finger touch on those devices. So since it's better
to target all at once, I'll just change the interface accordingly.

That might be a thing to remember for other pygbaggers out there.
